src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -68,20 +70,35 @@ fun subscribe(subscription: String, receiver: CompilerFleetMessageReceiver) {
     }
 }
 
+interface CompilerApi {
+    fun compile(request: CompileRequest): CompileResponse
+}
+
+class Compiler(address: String) : CompilerApi {
```

Comment:
Rename to TexbeCompilerImpl ? Because the difference between words Compiler and CompilerApi is negligible, while this class is in fact a concrete implementation

Reply:
Done.